lib/time_fetcher.js

```diff
@@ -91,13 +89,37 @@ async function hasTradeBeforeTime_CSGOFloat(hashCode, timestamp) {
     return false;
 }
 
-async function fetchListingTime_CSGOFloat(hashCode) {
-    const btn = document.querySelector(`#verify_${hashCode}_csgofloat`);
+async function fetchEnglishRow_CSGOFloat(index) {
+    let queryParams = location.search;
+    if (queryParams === '') {
+        queryParams = '?l=english';
+    } else {
+        queryParams += '&l=english';
```

Comment:
What if they already have a language in their query? Wouldn't this result in something like ...&l=french&l=english

Reply:
Sure, but the configurable language parameter is not meant for regular folks and I don't think there's a way to do it as a user within Steam.